app/views/shared/_top_navbar.html.erb

```diff
@@ -7,7 +7,13 @@
     <% end %>
 
     <div class="header-links" id="login_id">
-        <%= link_to "Feedback", "http://searchworks.stanford.edu/feedback", target: '_blank', rel: 'noopener noreferrer'  %>
+      <% if current_user != nil && current_user.webauth_user? %>
```

Comment:
Ah, I see you have to do this because we dropped the Null/Anonymous user all-together.  That's cool, although depending on how much we deal w/ the user object we may want to build that back in down the road.

In the meantime, this could be more sussinctly written as `if current_user && current_user.webauth_user?`.

If you find that you are doing the whole `if current_user && ...` frequently around the app, it may be a good idea to re-implement some sort of Null/Anonymous user object.